cvat/apps/dataset_manager/task.py

```diff
@@ -1006,19 +1042,21 @@ def get_job_data(pk):
 
     return annotation.data
 
+
 @silk_profile(name="POST job data")
 @transaction.atomic
-def put_job_data(pk, data):
-    annotation = JobAnnotation(pk)
+def put_job_data(pk, data, db_task: models.Task | None = None):
```

Comment:
```suggestion
def put_job_data(pk, data: AnnotationIR | dict, db_task: models.Task | None = None):
```

As the line was changed, we need to add annotation.

Reply:
done